src/popup.js

```diff
@@ -1,15 +1,19 @@
 let quickview = document.getElementById('quickview');
 let calculation = document.getElementById('calculation');
 let liveModification = document.getElementById('liveModification');
+let percentageColumn = document.getElementById('percentageColumn');
+let settings;
```

Comment:
```suggestion
```
No need to be a global variable, `settings` is only used within the function scope.